wolox-books/src/app/screens/book/components/detail/detail.component.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../../../colors';
+
+.container {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+.back {
+  margin-top: 48px;
+}
+
+.back-text {
+  font-size: 20px;
+  line-height: 24px;
+  margin-left: 12px;
+}
+
+.card {
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $soft-black;
+  display: flex;
+  height: 416px;
+  margin-top: 108px;
+  max-width: 900px;
+  padding: 24px;
+  position: relative;
+  width: 900px;
```

Comment:
width: 100%

Reply:
This is the size of the card, if I left with 100%, the card reduce its size to the content of the card